igvc_platform/launch/sensors.launch

```diff
@@ -22,10 +22,14 @@
     <include file="$(find igvc_platform)/launch/velodyne_lidar.launch" />
 
     <!-- Bottom IMU -->
-    <include file="$(find igvc_platform)/launch/imu_bottom.launch" />
+    <include file="$(find igvc_platform)/launch/imu_bottom.launch" >
+        <arg name="calibrate_imu" value="false"/>
```

Comment:
The argument "calibrate_imu" is already set to false in the imu_bottom.launch. Adding it again is kind of unnessary repetition. I think it would be better to have it only in one spot, so if it is changed, the values actually take effect.

Reply:
This is intentional, the default value in `imu_bottom` and `imu_top` shouldn't be changed, but we may have to set the value to true in `sensors.launch` if we decide to recalibrate every time. This allows quick changes and maintains a more object-like structure. `simulation.launch` follows a similar structure.